share/man/man7/arch.7

```diff
@@ -51,10 +51,11 @@ and later, unless otherwise noted.
 uses a flat address space.
 Variables of types
 .Vt unsigned long ,
-.Vt uintptr_t ,
+.Vt ptraddr_t ,
+.Vt ptrdiff_t ,
```

Comment:
I wouldn't include this otherwise ssize_t should be here too... upstream has decided to only do the unsigned ones

Reply:
Perhaps something to fix upstream then, but not in this local diff.